SU-SWS/saa_victoria_subtheme
# ADAPT-476: Adjusted selector for media precedes the card contents

# READY FOR REVIEW

# Summary
- makes selector work in more cases to fix padding below an icon. 
when on the sites they inserted the media, it gave a div without a class 'contextual region' and on my local it always has contextual region class, so i changed this to div, which is more reliable.

# Review By (Date)
- 7/21

# Urgency
- important!!

# Steps to Test

1. pull down branch
1. clear cache
2. test that spacing below icon is correct - you should see no padding on the top of card_contents when there is an icon. (and the spacing below the icon is provided from the icon). when there isn't an icon, you should see the regular padding, even around each side.

This is the problem (padding on top of card__contents and below the icon:
![image](https://user-images.githubusercontent.com/12898896/88097705-ce2dfb00-cb55-11ea-9616-99d6e9b4499c.png)

This is correct / or see figma
![image](https://user-images.githubusercontent.com/12898896/88097776-f0277d80-cb55-11ea-9a28-57b7f133774a.png)


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
